altos-rust/altos-core/src/test.rs

```diff
@@ -51,12 +51,6 @@ pub fn current_task() -> Option<&'static mut TaskControl> {
     unsafe { CURRENT_TASK.as_mut().map(|task| &mut ***task) }
 }
 
-pub fn create_two_tasks() -> (TaskHandle, TaskHandle) {
```

Comment:
Yeah that's 100% my bad, I was actually getting a warning flag that, the two task was an unused function a while ago.